package.json

```
@@ -183,11 +180,11 @@
     "react-16": "npm:react@16.14.0",
     "react-17": "npm:react@17.0.2",
     "react-dom": "18.2.0",
-    "react-dom-16": "npm:react-dom@16.14.0",
-    "react-dom-17": "npm:react-dom@17.0.2",
+    "react-dom-16": "npm:react-dom@^16.14.0",
+    "react-dom-17": "npm:react-dom@^17.0.2",
```

Based on our renovate configuration dev-deps are pinned.

```suggestion
    "react-dom-16": "npm:react-dom@16.14.0",
    "react-dom-17": "npm:react-dom@17.0.2",
```

---

Good catch, thanks

---

I'll do the fix and merge the PR :slightly_smiling_face: